examples/composite_queries/src/canister1/canister1.py

```diff
@@ -0,0 +1,167 @@
+from kybra import Async, CanisterResult, composite, Principal, query, update
+from src.canister1.types import Canister1, IntQueryResult, StringQueryResult
+from src.canister2.types import Canister2
+
+
+canister1 = Canister1(Principal.from_str('rrkah-fqaaa-aaaaa-aaaaq-cai'))
+canister2 = Canister2(Principal.from_str('ryjl3-tyaaa-aaaaa-aaaba-cai'))
+counter: int = 0
+
+
+# Composite query calling a query
+@query
+@composite
```

Comment:
Let's consider getting rid of the decorator and just making all async queries composite queries.